### ISSUE TITLE
Packages "mupdf" and "mupdf-tools" appear to be redundant

### ISSUE THREAD

**Issue by pimlottc** [Homebrew/homebrew-core#127361]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
HOMEBREW_VERSION: 4.0.10
ORIGIN: https://github.com/Homebrew/brew
HEAD: cf6614b50eac1c9f4bf386dc6989f0d443148df8
Last commit: 6 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: cf43a01aa352a63fd3a326a020143fd0650e96f4
Core tap last commit: 6 weeks ago
Core tap branch: master
Core tap JSON: 02 Apr 18:26 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.39.0 => /opt/homebrew/bin/git
Curl: 7.86.0 => /usr/bin/curl
macOS: 13.2.1-arm64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
Rosetta 2: false
```
### Verification

- [X] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Install [MuPDF](https://mupdf.com/)

### What happened (include all command output)?

There are two separate packages for MuPDF, `mupdf` and `mupdf-tools`.  The package descriptions are identical so it is not clear which one to choose.

```
% brew search mupdf
==> Formulae
mupdf                             mupdf-tools                       pymupdf                           mpd

% brew info mupdf
bre==> mupdf: stable 1.21.1 (bottled), HEAD
Lightweight PDF and XPS viewer
https://mupdf.com/
Conflicts with:
  mupdf-tools (because mupdf and mupdf-tools install the same binaries)
/opt/homebrew/Cellar/mupdf/1.21.1 (135 files, 48.5MB) *
  Poured from bottle using the formulae.brew.sh API on 2023-04-02 at 13:27:16
wFrom: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/mupdf.rb
License: AGPL-3.0-or-later
==> Dependencies
Build: lzip ✘, lzip ✘, pkg-config ✔
Required: freetype ✔, gumbo-parser ✔, harfbuzz ✔, jbig2dec ✔, jpeg-turbo ✔, mujs ✔, openjpeg ✔, openssl@1.1 ✔
==> Options
--HEAD
	Install HEAD version
 ==> Analytics
install: 42 (30 days), 1,193 (90 days), 13,847 (365 days)
install-on-request: 37 (30 days), 844 (90 days), 11,628 (365 days)
build-error: 0 (30 days)

% brew info mupdf-tools
==> mupdf-tools: stable 1.21.1 (bottled), HEAD
Lightweight PDF and XPS viewer
https://mupdf.com/
Conflicts with:
  mupdf (because mupdf and mupdf-tools install the same binaries)
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/mupdf-tools.rb
License: AGPL-3.0-or-later
==> Dependencies
Build: lzip ✘, lzip ✘
==> Options
--HEAD
	Install HEAD version
==> Analytics
install: 7 (30 days), 708 (90 days), 6,366 (365 days)
install-on-request: 7 (30 days), 708 (90 days), 6,346 (365 days)
build-error: 0 (30 days)
```

### What did you expect to happen?

I expected either a single mupdf package, OR meaningful descriptions that distinguish the two different packages.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew search mupdf
brew info mupdf
brew info mupdf-tools
```

**Comment by pimlottc** [comment]
Hmm, I *think* the difference might be that `mupdf` includes the libmupdf library while `mupdf-tools` only includes the command line tools.

I'm not sure why this is worth the bother of having two package, it doesn't seem like the dependencies for libmupdf are that heavy.  But if you really did want to split it out, wouldn't it make more sense to have a `libmupdf` package?  I'm not sure what Homebrew's standards are for library packages though.

**Comment by cho-m** [comment]
`mupdf-tools` introduced in https://github.com/Homebrew/homebrew-core/commit/b3b83a803a7 with original reason from https://github.com/Homebrew/legacy-homebrew/pull/36602
> mupdf tools (namely mutool/mudraw) and libmupdf does not depend on X11 and it's useful to have it installed without X11/homebrew-x11.

---

Since then, `mupdf` had some changes like:
- original X11 dependency was replaced by Homebrew's `freeglut`/`mesa` in https://github.com/Homebrew/homebrew-core/commit/9819c5ca10c4f748ab1e8f25f026e3009af3a01f
- and then these along with `libx11` were made Linux-only in https://github.com/Homebrew/homebrew-core/commit/777293377c4. On macOS, it now links to macOS system copies of GLUT/OpenGL.framework.

---

`mupdf-tools` is still a lighter-weight option, but exact reason may be a bit outdated.

`mupdf-tools` gets installs so may be useful to some users.
